code/src/index.css

```diff
@@ -5,9 +5,154 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url("/roses-background.jpg");
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  font-size: 40px;
+  margin-top: 100px;
+}
```

Comment:
in larger project its always a good practice to have classes on all styling components👩‍🏫